### ISSUE TITLE
Parameter.schema is ignored

### ISSUE THREAD

**Issue by antechrestos** [springdoc/springdoc-openapi#51]
Hello,

While trying to specify the expected type in the `@Parameter`with the `scheme` attribute, I see that it is ignored in the [following code](https://github.com/springdoc/springdoc-openapi/blob/3bafb411a661406728abb0be288fa8534461b1d1/springdoc-openapi-common/src/main/java/org/springdoc/core/ParameterBuilder.java#L89)

Why?

I was doing this:

```kotlin
fun list(   @Parameter(name = "trackerId", `in` = ParameterIn.PATH, required = true, schema = Schema(type = "string", example = "the-tracker-id"))
            @PathVariable trackerId: String,
            @Parameter(name = "start", `in` = ParameterIn.QUERY, required = false, schema = Schema(type = "string", format = "date-time", required = false, example = "1970-01-01T00:00:00.000Z"))
            @RequestParam(value = "start", required = false) startDate: Instant?,
            @Parameter(name = "end", `in` = ParameterIn.QUERY, required = false, schema = Schema(type = "string", format = "date-time", required = false, example = "1970-01-01T00:10:00.000Z"))
            @RequestParam(value = "end", required = false) endDate: Instant?): Collection<Result> {
...
```

And the output in json is 

```
{"name":"start","in":"query","required":false,"schema":{"type":"integer","format":"int64"}},{"name":"end","in":"query","required":false,"schema":{"type":"integer","format":"int64"}}
```

Thank you

**Comment by springdoc** [comment]
Hi, 

 The fix for this issue, will be available on v1.1.25 of springdoc-openapi.

**Comment by antechrestos** [comment]
@springdoc thanks!

I submitted a [pull request](https://github.com/springdoc/springdoc-openapi/pull/56) as the change you made may lead to a `null` schema.
